### PR TITLE
Jetbrains apps: Add RC and EAP packages

### PR DESCRIPTION
**Pull request by issaclin32**
This adds RC (release condidate) and EAP (early access program) of **JetBrains** apps to the bucket.

Notes:
* I have (1) run autoupdate locally, (2) installed, and (3) run the software of all these packages, to make sure they work properly.

### CODE REVIEW

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/webstorm-eap.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "IDE\\bin\\idea.properties",
+        "profile"
+    ],
+    "checkver": {
+        "url": "https://data.services.jetbrains.com/products/releases?code=WS&latest=true&platform=zip&type=eap",
+        "regex": "version\":\"(?<ver>[\\d.]+)\".*\"build\":\"(?<build>[\\d.]+)\"",
+        "replace": "${ver}-${build}"
+    },
+    "autoupdate": {
+        "url": "https://download.jetbrains.com/webstorm/WebStorm-$preReleaseVersion.exe#/cosi.7z",
```
Comment (rashil2000) [inline bucket/webstorm-eap.json:53]:
```suggestion
        "url": "https://download.jetbrains.com/webstorm/WebStorm-$preReleaseVersion.exe#/dl.7z",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/rider-eap.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    "JetBrains\\Rider"
+                ]
+            ]
+        }
+    },
+    "extract_to": "IDE",
+    "installer": {
+        "script": [
+            "& \"$bucketsdir\\extras\\scripts\\jetbrains\\portable.ps1\" $dir $persist_dir"
+        ]
```
Comment (rashil2000) [inline bucket/rider-eap.json:31]:
```suggestion
        "script": "& \"$bucketsdir\\extras\\scripts\\jetbrains\\portable.ps1\" $dir $persist_dir"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
One last thing: on the rare chance that someone doesn't have Extras bucket installed, the installation of all IDEs will fail. We should copy the `portable.ps1` script to Versions bucket too, and reference that instead of the one in Extras.

**Comment by issaclin32** [comment]
> One last thing: on the rare chance that someone doesn't have Extras bucket installed, the installation of all IDEs will fail. We should copy the `portable.ps1` script to Versions bucket too, and reference that instead of the one in Extras.

Done. Thank you so much for taking your time to check this.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**